setup-env

```diff
@@ -130,10 +166,10 @@ fi
 # Create a new virtual environment for this project
 if ! pyenv virtualenv "${env_name}"; then
   cat << END_OF_LINE
-  An existing virtual environment named $env_name was found.  Either delete this
-  environment yourself or re-run with --force option to have it deleted.
+    An existing virtual environment named $env_name was found.  Either delete this
+    environment yourself or re-run with --force option to have it deleted.
 
-  pyenv virtualenv-delete ${env_name}
+    pyenv virtualenv-delete ${env_name}
 
```

Comment:
Is this a needless blank line?

Reply:
Yeah didn't see this until now but I have it fixed here https://github.com/cisagov/skeleton-generic/pull/166/commits/1861b9b86778613bdaeff804418f2d00706f4a08

Reply:
I still see the blank line.

Reply:
> I still see the blank line.

Sorry I thought you were talking about the unnecessary spacing. Which blank line are you talking about? I think the rest of the script should be like it was originally but I am probably just missing something

Reply:
Line 157, but maybe that line is required.

Reply:
I think it could be removed. 🤷

 @felddy any issue with that? I believe you were the original author.

Reply:
Nevermind, this is unimportant.